### PR TITLE
Partial online multiplayer game implementation

### PR DESCRIPTION
**Pull request by laurislopata**
In this PR I started to implement the online buzzer game. For now, I synced up user connection to one party to demonstrate that it prints their emails on the screen. Will continue working on this next sprint

### CODE REVIEW

File: app/src/main/res/layout/activity_choose_party_room.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    android:background="@color/floral_white"
+    tools:context=".activity.ChoosePartyRoomActivity">
+
+    <EditText
+        android:id="@+id/roomNameInput"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="440dp"
+        android:ems="10"
+        android:hint="Party Name"
```
Comment (Tsathogguaa) [inline app/src/main/res/layout/activity_choose_party_room.xml:16]: Hardcoded string here. It's fine overall, but please remember to fix it in your next PR

**Review by Tsathogguaa — APPROVED** [review]
Great work! There was just one comment I made, but it's not at a level where it requires an instant change. Good job!

File: app/src/main/res/layout/activity_choose_party_room.xml (hunk 1 above)
Comment (laurislopata) [reply app/src/main/res/layout/activity_choose_party_room.xml:16]: Noted, I will fix it in the next PR

File: app/src/main/java/ch/sdp/vibester/activity/ChoosePartyRoomActivity.kt [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package ch.sdp.vibester.activity
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import android.widget.EditText
+import ch.sdp.vibester.R
+
```
Comment (MaximeZmt) [inline app/src/main/java/ch/sdp/vibester/activity/ChoosePartyRoomActivity.kt:9]:
```suggestion
/**
* What is the purpose of the class ???
*/
```

File: app/src/main/java/ch/sdp/vibester/activity/PartyRoomActivity.kt [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import ch.sdp.vibester.helper.PartyRoom
+import ch.sdp.vibester.user.User
+import com.google.firebase.database.DataSnapshot
+import com.google.firebase.database.DatabaseError
+import com.google.firebase.database.ValueEventListener
+import dagger.hilt.android.AndroidEntryPoint
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class PartyRoomActivity : AppCompatActivity() {
```
Comment (MaximeZmt) [inline app/src/main/java/ch/sdp/vibester/activity/PartyRoomActivity.kt:20]:
```suggestion
/**
* What is class purpose
*/
@AndroidEntryPoint
class PartyRoomActivity : AppCompatActivity() {
```

File: app/src/main/java/ch/sdp/vibester/helper/PartyRoom.kt [hunk 4]
```diff
@@ -0,0 +1,36 @@
+package ch.sdp.vibester.helper
+
+
+class PartyRoom() {
```
Comment (MaximeZmt) [inline app/src/main/java/ch/sdp/vibester/helper/PartyRoom.kt:4]:
```suggestion
/**
* DOCSSSS ? 
*/
class PartyRoom() {
```

File: app/src/main/java/ch/sdp/vibester/helper/PartyRoom.kt [hunk 5]
```diff
@@ -0,0 +1,36 @@
+package ch.sdp.vibester.helper
+
+
```
Comment (MaximeZmt) [inline app/src/main/java/ch/sdp/vibester/helper/PartyRoom.kt:3]: Maybe make it serializable

**Review by MaximeZmt — APPROVED** [review]
Approved because it's late in the week and you can address the issues next week.
Good work 👍

**Comment by laurislopata** [comment]
Thank you for the comments, I'll add it in the next PR

**Pull request merged**